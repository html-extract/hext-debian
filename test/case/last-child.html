<div class="first-div">
  <ul>
    <li class="7 first-li"></li>
    <li class="6 nope"></li>
    <li class="5 nope"></li>
    <li class="4 nope"></li>
    text
    <li class="3 nope"></li>
    text
    <li class="2 nope"></li>
    text
    <li class="1 last-li"></li>
  </ul>
</div>
