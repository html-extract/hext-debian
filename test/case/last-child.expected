{"class":"1 last-li"}
{"class":"first-div"}
